alleycats-core/src/main/scala-2.13+/alleycats/compat/IterableEmptyInstance.scala

```diff
@@ -25,6 +25,6 @@ package compat
 import collection.Factory
 
 abstract class IterableEmptyInstance {
-  implicit def iterableIsEmpty[CC[X] <: Iterable[X], A](implicit factory: Factory[A, CC[A]]): Empty[CC[A]] =
+  implicit def iterableIsEmpty[CC[X] <: IterableOnce[X], A](implicit factory: Factory[A, CC[A]]): Empty[CC[A]] =
```

Comment:
I don't think we want this change. IterableOnce may be mutable, and we generally don't make instances for those types.

Reply:
Thx, didn't think of that.